index.js

```diff
@@ -30,16 +41,11 @@ async function initialize(projectId, options) {
   pId = projectId;
 
   firebase.initializeApp(firebaseConfig);
-  const database = firebase.database();
+  database = firebase.database();
 
   currVersion = localStorage.getItem('jellySyncVersion');
-  endpoint = localStorage.getItem('jellysyncEndpoint');
-
-  if (!endpoint) {
-    endpoint = await getEndpoint(projectId);
-  }
 
-  dbRef = database.ref(`project/${endpoint}`);
+  dbRef = database.ref(`/projects/${projectId}/${endpoint.id}`);
```

Comment:
isnt it possible for endpoint to be undefined here?

Reply:
what should we do in that case?

Reply:
1. we should put retry logic into axios (npm axios-retry)
2. for now check for an empty value and bail gracefully